packages/@coorpacademy-components/src/atom/battle-opponent/style.css

```diff
@@ -112,6 +112,18 @@
   align-items: center;
 }
 
+.toolTipWrapper {
+  position: absolute;
+  width: 100%;
+}
+
+.card .tooltipIconContainer + .toolTipWrapper [class*="__react_component_tooltip"]:not([class*="toolTipReact"]) {
+  position: sticky;
+  top: unset !important;
+  left: unset !important;
+  max-width: 75%;
```

Comment:
to fix external tooltip lib's issue of keyboard navigation positioning on key only events